JotunnLib/Managers/MockManager.cs

```diff
@@ -82,16 +82,13 @@ private void RemoveMockPrefabs(On.ObjectDB.orig_Awake orig, ObjectDB self)
         {
             orig(self);
 
-            if (SceneManager.GetActiveScene().name == "main")
+            if (SceneManager.GetActiveScene().name == "main" && MockPrefabContainer.transform.childCount > 0)
```

Comment:
Is not necessary, foreach() does not enumerate where there is none. Or is there any special intent?

Reply:
I have no idea when or why I did this, I'll remove it :P